userguide/jails.rst

```diff
@@ -759,17 +727,18 @@ entries available for a jail.
 Accessing a Jail Using SSH
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Accessing a jail is not limited to using Jail Add, the Jail Wizard, or
-using one of the Shells that are built in to the UI. Enable the ssh
-daemon `sshd(8)
-<https://www.freebsd.org/cgi/man.cgi?query=sshd&apropos=0&sektion=8>`__
-in a jail, create a user account to log in with, and ssh access to
-the jail via any computer that has network access to the %brand% server
-is now possible.
 
-Log in to a shell to get :guilabel:`cli` access to the jail to enable SSH
-on. To use the Jails Shell click |ui-options| --> :guilabel:`Shell` on
-the desired jail. A root shell will open as shown in this example:
+The ssh daemon
+`sshd(8) <https://www.freebsd.org/cgi/man.cgi?query=sshd>`__
+must be enabled to access a jail from another system using SSH.
+
+Access the shell of the jail in the %brand% UI to enable SSH. Log into
```

Comment:
This sentence is redundant, it's just stating a part of what is about to be done below.  Better without it.